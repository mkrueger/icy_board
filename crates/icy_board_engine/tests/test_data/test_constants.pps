PRINT AUTO,","
PRINT BELL,","
PRINT DEFS,","
PRINT ECHODOTS,","
PRINT ERASELINE,","
PRINT FCL,","
PRINT FIELDLEN,","
PRINT FNS,","
PRINT F_EXP,","
PRINT F_MW,","
PRINT F_REG,","
PRINT F_SEL,","
PRINT F_SYS,","
PRINT GRAPH,","
PRINT GUIDE,","
PRINT HIGHASCII,","
PRINT LANG,","
PRINT LFAFTER,","
PRINT LFBEFORE,","
PRINT LOGIT,","
PRINT LOGITLEFT,","
PRINT NC,","
PRINT NEWLINE,","
PRINT NOCLEAR,","
PRINT O_RD,","
PRINT O_RW,","
PRINT O_WR,","
PRINT SEC,","
PRINT SEEK_CUR,","
PRINT SEEK_END,","
PRINT SEEK_SET,","
PRINT STACKED,","
PRINT S_DB,","
PRINT S_DN,","
PRINT S_DR,","
PRINT S_DW,","
PRINT UPCASE,","
PRINT WORDWRAP,","
PRINT YESNO,","
PRINT START_BAL,","
PRINT START_SESSION,","
PRINT DEB_CALL,","
PRINT DEB_TIME,","
PRINT DEB_MSGREAD,","
PRINT DEB_MSGCAP,","
PRINT DEB_MSGWRITE,","
PRINT DEB_MSGECHOED,","
PRINT DEB_MSGPRIVATE,","
PRINT DEB_DOWNFILE,","
PRINT DEB_DOWNBYTES,","
PRINT DEB_CHAT,","
PRINT DEB_TPU,","
PRINT DEB_SPECIAL,","
PRINT CRED_UPFILE,","
PRINT CRED_UPBYTES,","
PRINT CRED_SPECIAL,","
PRINT SEC_DROP,","