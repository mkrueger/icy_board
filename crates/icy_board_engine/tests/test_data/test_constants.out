8192,2048,0,1,32,2,2,1,2,16,1,4,8,1,4,4096,4,256,128,32768,65536,0,64,1024,0,2,1,2,1,2,0,16,3,0,1,2,8,512,16384,0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,